source/val/validate_annotation.cpp

```diff
@@ -52,6 +52,16 @@ bool IsValidWebGPUDecoration(uint32_t decoration) {
   }
 }
 
+bool IsValidVulkanDecoration(uint32_t decoration) {
```

Comment:
Better to just inline this. The name makes me think it has a broader scope than how it has been implemented.

Reply:
The `IsValidWebGPUDecoration` is right above it, was trying to be consistent, should I just inline both then?

Reply:
Leave the WebGPU one for now. I think it needs some broader cleanup anyways.